tests/playwright/src/model/pages/deploy-to-kubernetes-page.ts

```diff
@@ -0,0 +1,41 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+
+import { BasePage } from '../pages/base-page';
+
+export class DeployToKubernetesPage extends BasePage {
+  readonly deployButton: Locator;
+  readonly doneButton: Locator;
+  readonly podNameField: Locator;
+  constructor(page: Page) {
+    super(page);
+    this.podNameField = this.page.locator('#podName');
```

Comment:
We need to add attribute for this locator.

Reply:
I would say we need to add more than just one things here. Whole page could make use of some ARIA functionality revision.